packages/libretls/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://git.causal.agency/libretls/about/
+TERMUX_PKG_DESCRIPTION="libtls for OpenSSL"
+TERMUX_PKG_LICENSE="ISC"
+# There's no license file in the tarball source
+# Instead, copy README.7 as it also includes what LICENSE that they use.
+TERMUX_PKG_LICENSE_FILE="README.7"
```

Comment:
`README.7` is not qualified as `TERMUX_PKG_LICENSE_FILE` for `ISC` license as it does not contain copyright information.

Reply:
See also https://github.com/termux/termux-packages/pull/9623#discussion_r837710469.

Reply:
so remove that line?

Reply:
Yep. And then prepare a custom `LICENSE` file in the source file tree.